src/I18n/Exception/CannotFormat.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @copyright     Copyright (c) 2017-2020, Aura for PHP
```

Comment:
Is this copyright tag acceptable? Ref https://github.com/auraphp/Aura.Intl/blob/3.x/LICENSE#L3

Reply:
```suggestion
 * @copyright     Copyright (c) 2017, Aura for PHP
```
No changes after 2017, so 2020 is inaccurate, especially if range 2017-2020 specified

Reply:
A range isn't necessary for copyright. We can just declare the date it was changed.